tests/bofire/surrogates/test_gps.py

```diff
@@ -321,6 +361,45 @@ def test_MixedSingleTaskGPHyperconfig():
         )
 
 
+def test_MultiTask_input_preprocessing():
+    # test that if wrong encoding is used, there is an error
```

Comment:
you can move this tests for the errors into the specs firectoy via `add_invalid`